assignment.js

```diff
@@ -113,6 +131,38 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+let playerAChoice = "scissors"
+let playerBChoice = "paper"
+let outcome = ""
+
+if (playerAChoice == "rock") {
```

Comment:
@justfinish nice approach covering all of the possible outcomes. 

The approach that you're using involves nested `if` statements and while it works, you will want to avoid nesting if possible. The reason is that nesting can add complexity to your code which makes things more challenging to debug. In this case, the code was still easy to follow.